.github/workflows/lint.yaml

```diff
@@ -0,0 +1,42 @@
+name: Lint
+
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+
+jobs:
+
+    lint:
```

Comment:
change to "solidity" to show that we're linting solidity files (and JSON stuff too I guess)